scripts/travis_install

```diff
@@ -40,6 +40,7 @@ conda install --yes abstract-rendering
 conda install --yes multiuserblazeserver pillow
 ##hack - we need latest blaze right now
 conda install --yes -c blaze blaze
+pip install git+https://github.com/birdsarah/pytest-selenium.git
```

Comment:
Yes indeed. I want to use the maintained package not my fork, and I think I just got the answer to make that possible.  Hopefully @damianavila will be able to help get it onto bokeh channel then.